src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -685,14 +685,31 @@ internal IEnumerable<PortModel> GetGroupOutPorts(IEnumerable<NodeModel> ownerNod
                 );
         }
 
-        private List<ProxyPortViewModel> CreateProxyPorts(IEnumerable<PortModel> groupPortModels)
+        private List<PortViewModel> CreateProxyInPorts(IEnumerable<PortModel> groupPortModels)
         {
             var originalPortViewModels = WorkspaceViewModel.Nodes
-                .SelectMany(x => x.InPorts.Concat(x.OutPorts))
+                .SelectMany(x => x.InPorts)
+                .Where(x => groupPortModels.Contains(x.PortModel))
+                .ToList();
+
+            var newPortViewModels = new List<PortViewModel>();
+            for (int i = 0; i < groupPortModels.Count(); i++)
+            {
+                var model = groupPortModels.ElementAt(i);
+                newPortViewModels.Add(originalPortViewModels[i].CreateProxyPortViewModel(model));
+            }
+
+            return newPortViewModels;
+        }
+
+        private List<PortViewModel> CreateProxyOutPorts(IEnumerable<PortModel> groupPortModels)
```

Comment:
same question here.

Reply:
@SHKnudsen Do you know?

Reply:
There where only one method before to create proxyports, so can't say for sure about this one. But the original method was called when the group is collapsedsed